### ISSUE TITLE
after merge

### ISSUE THREAD

**Issue by Ericalvin** [ethereum/go-ethereum#25764]
Hi, i use geth + prysm , but now after merge, geth log say 

`Sep 15 07:10:43 ip-10-218-25-100 geth[996]: WARN [09-15|07:10:43.146] Local chain is post-merge, waiting for beacon client sync switch-over..`

and prysm log say 
`[2022-09-15 07:14:08]  WARN powchain: Execution client is not syncing`
`[2022-09-15 07:14:08]  INFO initial-sync: Processing block batch of size 63 starting from  0x504d1c2c... 1684114/4700168 - estimated time remaining 37h34m9s blocksPerSecond=22.3 peers=44`

is this because my prysm not sync finish?

**Comment by MariusVanDerWijden** [comment]
Yes, looks like prysm needs more time. You can try using checkpoint sync on prysm

**Comment by Ericalvin** [comment]
@MariusVanDerWijden  thank you, i will wait for prysm sync finish, and see what will say.

**Comment by lijie9703** [comment]
Please do it after merging, should I use prysm or geth for node operations, transfers, query transaction hashes, etc.

**Comment by Sreeram1993** [comment]
@MariusVanDerWijden  My geth archival node had to be restarted after merge due to some unforeseen reasons. I hope it would still sync after restart ?

**Comment by MariusVanDerWijden** [comment]
yes it will

**Comment by Sreeram1993** [comment]
Can anyone help me with the total chain data size of beacon chain ? Archive node I can see from beacon scan that it is 3.5 TB+

**Comment by begetan** [comment]
@Sreeram1993 Archive node is about 13 TB, and full node is about 700 GB for the fresh sync.

**Comment by Sreeram1993** [comment]
> @Sreeram1993 Archive node is about 13 TB, and full node is about 700 GB for the fresh sync.
Perfect, thanks.
